### DISCUSSION TITLE
请不要下载这个文件！

### DISCUSSION THREAD

**Discussion by hejiehao** [Meloong-Git/PCL#4597] [讨论]
最近在 GitHub 上出现了如下的 Comment

> Download<病毒链接>
password: changeme
In the installer menu, select "gcc."

> to fix your trouble try download this fix, i see it in another issue,
<病毒链接>
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

如果你看见了类似这样的 Comment，请不要下载！这是病毒！

**Comment by lisonge** [comment]
Currently, this virus has led to the theft of GitHub accounts, resulting in approximately 4,000 comments containing virus files being posted.

<https://github.com/search?q=%22password%3A+changeme%22&type=issues>

Many repositories' issues have been affected.

**Reply by n0099** [reply]
https://v2ex.com/t/1067998

**Reply by luduting** [reply]
If you click on this link, your GitHub account will be hacked

**Comment by wuliaodexiaoluo** [comment]
已向 MediaFire 反馈此问题

**Comment by SALTWOOD** [comment]
![QQ_1724686537788](https://github.com/user-attachments/assets/4b6ad195-8de3-48bb-b8fe-d6bfe466c854)
已经通过 GitHub Support 举报。以下是我通过gpt生成的一篇短文，大家可以参考：

Dear GitHub Support Team,

I hope this message finds you well. I am writing to report a serious issue involving a user named "Superming997" who has been spreading malicious content across multiple repositories on GitHub. This user has been posting comments that include a link to a file hosted on MediaFire, which appears to contain a virus.

The comments have been found in the following repositories:

[HMCL-dev/HMCL#3267](https://github.com/HMCL-dev/HMCL/issues/3267#issuecomment)
[HMCL-dev/HMCL#3266](https://github.com/HMCL-dev/HMCL/issues/3266#issuecomment)
[microsoft/vscode-remote-release#10189](https://github.com/microsoft/vscode-remote-release/issues/10189)
In the comments, the user provides a download link to a file named "fix.zip" with a password and instructs users to select "gcc" in the installer menu. This is a clear attempt to trick users into downloading and executing malicious software.

For your reference, here is the direct link to the file:
**Hidden**

Please take immediate action to investigate and remove this content, as it poses a significant risk to the GitHub community. Additionally, I recommend taking appropriate measures against the user "Superming997" to prevent further distribution of harmful files.

Thank you for your prompt attention to this matter.

Best regards,
[Your Name]

**Comment by wyc-26** [comment]
转到 #4616 继续讨论（
